### ISSUE TITLE
WCAG 2 Layers of Guidance still refers to WCAG 2.0

### ISSUE THREAD

**Issue by michael-n-cooper** [w3c/wcag#411]
_From @jake-abma on June 8, 2018 15:26_

At [https://www.w3.org/TR/WCAG21/#wcag-2-layers-of-guidance](url) I see two references to WCAG 2.0.

Shouldn't this be adjusted to 2.1?

> Success Criteria - For each guideline, testable success criteria are provided to allow **WCAG 2.0** to be used

> Sufficient and Advisory Techniques - For each of the guidelines and success criteria in the **WCAG 2.0** document itself,

And same for the following link which goes to a 2.1 URL: 

> Where common failures are known, these are also documented. [ See also Sufficient and Advisory Techniques in Understanding WCAG 2.0.](url).

_Copied from original issue: w3c/wcag21#957_

**Comment by michael-n-cooper** [comment]
_From @jake-abma on June 8, 2018 15:44_

another one can be seen at: https://www.w3.org/TR/WCAG21/#wcag-2-1-supporting-documents

> See Web Content Accessibility Guidelines (WCAG) Overview for a description of the WCAG 2.0 supporting material

**Comment by annethyme** [comment]
There is actually 2 mentions of WCAG 2.0 in https://www.w3.org/TR/WCAG21/#wcag-2-1-supporting-documents

**Comment by alastc** [comment]
In case anyone else comes here to report this, the incorrect instances of WCAG 2.0 have all been dealt with and can be seen in the editors draft:
https://w3c.github.io/wcag/21/guidelines/ 

They will appear in the officially published set when it is next published (should be soon after TPAC, late Oct 2018).

**Comment by fstrr** [comment]
Closing this as the issue is resolved.

**Comment by patrickhlauke** [comment]
@alastc 

> They will appear in the officially published set when it is next published (should be soon after TPAC, late Oct 2018).

noting that the published version on https://www.w3.org/TR/WCAG21/#wcag-2-1-supporting-documents is still from June 2018, so the incorrect references to "WCAG 2.0" are still on the published version as we speak.

**Comment by fstrr** [comment]
Sorry—shouldn't have closed this prematurely.

Looking in the current version of the [guidelines index page in GitHub](https://github.com/w3c/wcag/blob/main/guidelines/index.html), it looks like there's a 2.2 update ready to go for this which addresses those references to WCAG 2.0 in the Supporting Documents section:

- [Line 63](https://github.com/w3c/wcag/blob/main/guidelines/index.html#L63=) says "WCAG 2.2" instead of "WCAG 2.0"
- [Line 79](https://github.com/w3c/wcag/blob/main/guidelines/index.html#L78=) says "WCAG 2.2" instead of "WCAG 2.0"

Additionally, [Line 52](https://github.com/w3c/wcag/blob/main/guidelines/index.html#L52=) and [Line 55[(https://github.com/w3c/wcag/blob/main/guidelines/index.html#L55=) have been corrected to say "WCAG 2.2" instead of "WCAG 2.0".

**Comment by alastc** [comment]
@michael-n-cooper is the /TR/ page going to be updated? If not, we might be best changing it to "WCAG 2.x" for 2.2 and any subsequent versions.

**Comment by kfranqueiro** [comment]
Fixes for these instances appear to have since been published for both 2.1 and 2.2.
